src/utils/api/helper.ts

```diff
@@ -1025,9 +1026,10 @@ export const npgSessionsFields = async (
               | PaymentMethod
               | undefined
           )?.paymentMethodId || "";
-        return apiPaymentEcommerceClientWithRetry.createSession({
+        return apiPaymentEcommerceClientWithRetryV2.createSession({
```

Comment:
why introduce v2 for create session api call? there is no breaking changes here, the only addition is an optional header that is not a breaking changes

Reply:
i understand, but the usage of V2 apis on all changes related the new language header has been decided during the refinement of the tasks involved.